app/emoji.go

```diff
@@ -148,6 +148,22 @@ func (a *App) GetEmoji(emojiId string) (*model.Emoji, *model.AppError) {
 	}
 }
 
+func (a *App) GetEmojiByName(emojiName string) (*model.Emoji, *model.AppError) {
+	if !*a.Config().ServiceSettings.EnableCustomEmoji {
+		return nil, model.NewAppError("GetEmoji", "api.emoji.disabled.app_error", nil, "", http.StatusNotImplemented)
+	}
+
+	if len(*a.Config().FileSettings.DriverName) == 0 {
+		return nil, model.NewAppError("GetEmoji", "api.emoji.storage.app_error", nil, "", http.StatusNotImplemented)
+	}
```

Comment:
This case is implicitly tested in your `TestGetEmojiByName` test when you `CheckNoError` but not explicitly tested. If you didn't intend to test it then that's cool. I'm 0/5.